src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -18,6 +18,52 @@
       ]
     }
   },
+  {
+    "name": "AppService_FunctionApp_HTTPS",
+    "description": "Function App should only be accessible over HTTPS",
+    "recommendation": "Use HTTPS to ensure server/service authentication and protect data in transit from network layer eavesdropping attacks.",
```

Comment:
Other built in rule recommendations don't end with periods

Reply:
These were taken straight from Policy.  I wonder if Policy is inconsistent.